
package it.csi.myprod.myprodsrvqrk.api.wsclient.iride2pep;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.3.2
 * 2020-12-21T09:41:23.719+01:00
 * Generated source version: 3.3.2
 */

@WebFault(name = "fault6", targetNamespace = "http://pep.base.nmsf.csi.it/")
public class CertOutsideValidityException_Exception extends Exception {

    private it.csi.myprod.myprodsrvqrk.api.wsclient.iride2pep.CertOutsideValidityException fault6;

    public CertOutsideValidityException_Exception() {
        super();
    }

    public CertOutsideValidityException_Exception(String message) {
        super(message);
    }

    public CertOutsideValidityException_Exception(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public CertOutsideValidityException_Exception(String message, it.csi.myprod.myprodsrvqrk.api.wsclient.iride2pep.CertOutsideValidityException fault6) {
        super(message);
        this.fault6 = fault6;
    }

    public CertOutsideValidityException_Exception(String message, it.csi.myprod.myprodsrvqrk.api.wsclient.iride2pep.CertOutsideValidityException fault6, java.lang.Throwable cause) {
        super(message, cause);
        this.fault6 = fault6;
    }

    public it.csi.myprod.myprodsrvqrk.api.wsclient.iride2pep.CertOutsideValidityException getFaultInfo() {
        return this.fault6;
    }
}
